src/graph/entities/qg_edge.h

```diff
@@ -56,8 +57,8 @@ QGEdge *QGEdge_Clone
 	const QGEdge *orig
 );
 
-// determine whether this is a variable length edge
-bool QGEdge_VariableLength
+// determine whether this is a edge with a range of length defined (variable length [*minHops..maxHops] or fixed length [*Hops])
+bool QGEdge_RangeLength
```

Comment:
Please replace with the following 2 functions:
1. `QGEdge__VariableLength` which simply returns true if `minHops ` != `maxHops`
2.`QGEdge_RangeLength` which simply returns true if `minHops` > 1

Please make sure patterns such as `()-[*0]-()` are still supported and are functioning as expected.